client/gutenberg/extensions/publicize/connection.jsx

```diff
@@ -11,14 +11,36 @@
  * External dependencies
  */
 import { Component } from '@wordpress/element';
-import { Disabled, FormToggle } from '@wordpress/components';
+import { Disabled, FormToggle, Notice, ExternalLink } from '@wordpress/components';
 import { withSelect } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
 import PublicizeServiceIcon from './service-icon';
 
+/**
+ * If this is the Google+ connection, display a notice.
+ *
+ * @param {string} serviceName Name of the connnected social network.
+ * @returns {object} Message warning users of Google+ shutting down.
+ */
+const maybeDisplayGooglePlusNotice = serviceName =>
+	'google-plus' === serviceName && (
+		<Notice status="error">
```

Comment:
`Notice` component gives "X" icon for closing by default, but it doesn't do anything right now.

Can we just hide it by setting `isDismissible` false, if we don't intend to let users hide it? They'll just start contacting support otherwise. :-)

<img width="268" alt="image" src="https://user-images.githubusercontent.com/87168/51789587-cd035400-2192-11e9-930c-8b73fdedc3ba.png">